openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/explorePage.java

```diff
@@ -0,0 +1,179 @@
+package org.openmetadata.catalog.selenium.objectRepository;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class explorePage {
```

Comment:
Use `ExplorePage` instead of `explorePage` since the code is in Java?

Same comment for other classes in this PR